src/Atmos/Model/prog_prim_conversion.jl

```diff
@@ -0,0 +1,172 @@
+####
```

Comment:
Should this interface be updated for balance law first argument?

Reply:
Yes, will do